docs/content/stable/yugabyte-platform/alerts-monitoring/prometheus-setup.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Use your own Prometheus instance
+headerTitle: Use a custom Prometheus instance
+linkTitle: Custom Prometheus
+description: Export universe metrics to your own Prometheus instance
+headcontent: Export universe metrics to your own Prometheus instance
+menu:
+  stable_yugabyte-platform:
+    parent: alerts-monitoring
+    identifier: prometheus-setup
+    weight: 90
+type: docs
+---
+
+While YugabyteDB Anywhere includes an [embedded Prometheus instance](../anywhere-metrics/) and uses it for storing time-series metrics data, you can also set up your own independent and separately-managed Prometheus instance.
+
+This additional Prometheus instance can collect, visualize, alert on, and analyze universe metrics in your own observability tools. Prometheus itself, or any tool able to use Prometheus as a data source (such as Grafana) can be used.
+
+Your independent Prometheus instance will scrape data from database nodes directly (in the case of VM-based universes) or scrape data from a K8s Prometheus Service Monitor (in the case of K8s-based universes). This data scraping runs in parallel to and independently from the YugabyteDB Anywhere-embedded Prometheus.
+
+## Kubernetes
+
+YugabyteDB allows scraping of universe metrics in Kubernetes using a ServiceMonitor. Similar to a Pod or Deployment, ServiceMonitor is also a resource that is handled by the Prometheus Operator. The ServiceMonitor resource selects a Kubernetes Service based on a given set of labels. With the help of this selected Service, the Prometheus Operator creates scrape configurations for Prometheus. Prometheus uses these configurations to actually scrape the application pods.
+
+The ServiceMonitor resource can have details like port names or numbers, scrape interval, HTTP path where application is exposing the metrics, and so on. In the case of the `yugabytedb/yugabyte` Helm chart, it creates ServiceMonitors for YB-Master as well as YB-TServer pods.
+
+### Prerequisites
+
+Install the Prometheus Operator on your Kubernetes universe as follows:
```

Comment:
We also don't need to give directions to install prometheus operator, this is something every customer might do differently, these instructions are also not correct.